src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -56,7 +57,8 @@ public ProjectLaunchTargetsProvider(
         IVsUIService<SVsShellDebugger, IVsDebugger10> debugger,
         IRemoteDebuggerAuthenticationService remoteDebuggerAuthenticationService,
         Lazy<IProjectHotReloadSessionManager> hotReloadSessionManager,
-        Lazy<IHotReloadOptionService> debuggerSettings)
+        Lazy<IHotReloadOptionService> debuggerSettings,
+        Lazy<IActiveConfiguredValue<IProjectHotReloadLaunchProvider>> projectHotReloadLaunchProvider)
```

Comment:
I don't think `Lazy<>` helps here. It's fine to use `IActiveConfiguredValue<>` directly (it initializes itself lazily anyway) and avoid the (small) overlead of the `Lazy` here.